cmd/openshift-install/main.go

```diff
@@ -54,6 +55,8 @@ func installerMain() {
 		newVersionCmd(),
 		newGraphCmd(),
 		newCompletionCmd(),
+		azure.NewMigrateAzurePrivateDNSEligibleCmd(),
+		azure.NewMigrateAzurePrivateDNSMigrateCmd(),
```

Comment:
this puts these at the same level as the `create, destroy`.. that is not correct.
```
openshift-install migrate azure-privatedns-eligible
openshift-install migrate azure-privatedns
```

i'm sorry for not looking at this before, but i don't think we can keep these at root same as `create, destroy`.

Reply:
Fixed!